app/src/main/java/org/jellyfin/androidtv/util/Utils.java

```diff
@@ -206,6 +206,10 @@ public static PopupMenu createPopupMenu(Activity activity, View view, int gravit
         return new PopupMenu(activity, view, gravity);
     }
 
+    public static PopupMenu createPopupMenu(Context context, View view, int gravity) {
```

Comment:
I think the `createPopMenu` function which accepts an `Activity` instance can be removed now